nss-day-cohort-43-csharp/Fullstack-Rage-Against-The-Virtual-Machine
# Category edit and delete

Allows the editing and deleting of categories and addresses non admin bug on categories page

Testing:
- Run SQL queries again to create a database
- Change the UID's of your users to the UIDs in your firebase
- Log in as an admin
- Navigate to the categories tab
- Add a new Category
- Find that category on the list, edit the name, and save it
- Delete the category from the list
- Log in as a non admin and navigate to the "/categories" page. You should be redirected to the homepage
